packages/ui/components/ui/tx/actions-views/swap/one-way-swap.tsx

```diff
@@ -14,16 +14,32 @@ export const OneWaySwap = ({ input, output }: { input: ValueView; output: ValueV
   return (
     <div className='flex items-center justify-between'>
       <ValueViewComponent view={input} />
-      <div className='relative mx-2 flex items-center'>
-        <div className='h-px w-[40px] bg-gray-500' />
-        <div
-          className='absolute right-[-6px] size-0'
-          style={{
-            borderTop: '6px solid transparent',
-            borderBottom: '6px solid transparent',
-            borderLeft: '6px solid gray',
-          }}
-        />
+      <div className='relative mx-2 flex items-center justify-center'>
+        <div className='flex items-center justify-center w-16 h-6 rounded-full bg-gray-800 border border-gray-500'>
```

Comment:
updated arrow for one-way swaps

<img width="367" alt="Swap Claim Fee" src="https://github.com/user-attachments/assets/e492dce5-9fac-4e30-a6a3-2ad696fb9624">